pkg/telemetry/prometheus/node.go

```diff
@@ -12,7 +12,7 @@ import (
 
 const (
 	livekitNamespace    string = "livekit"
-	forceUpdateInterval int64  = 15
+	forceUpdateInterval int64  = 5
```

Comment:
This is going to produce 0 deltas. Telemetry stars worker produces data only every 10 seconds. Force updating sooner would produce zeros I think.

Reply:
ah gotcha.. yeesh.. I was trying to figure out a way to make it update sooner.. so we can reduce the amount of time that a session would be assigned to a dead node. this is pretty tricky.. let me think about it a bit more.